INFRASTRUCTURE_MENU.md

```diff
@@ -0,0 +1,61 @@
+# OpenJS Foundation Project Infrastructure Services
+
+The OpenJS Foundation provides works with a number of services to provide critical infrastructure services to hosted projects.  We expect projects to be respectful of these services, to abide by their terms of use, and to use the for the good of the project and the OpenJS Foundation.
+
+## Billing for services and mitigating the bus factor
+**For all project services, please add an OpenJS Foundation account at an owner or highest-level of permission access.**  This helps ensure continuity by reducing the bus factor on the project, and ensures you are never locked out.  It is also **required** in order for the OpenJS Foundation to pay service fees on behalf of your project.  Access to the OpenJS Foundation administrator/owner account will never be shared with others, and will only be granted to operations, IT, and finance staff at the Linux Foundation.
+
+If you don’t know the name of the OpenJS Foundation account for a service, please contact operations@openjsf.org.
+
+## Websites
+Digital Ocean offers free droplets to OpenJS Foundation projects.  Projects are solely responsible for the content and design of their websites.
+
+## Servers
+Additional servers needed to run scripts, bots, or other project applications may also be deployed on Digital Ocean.  These resources are generously provided by Digital Ocean for free, so please be respectful of the arrangement and do not deploy overly intensive workloads such as CI/CD without discussion with the Foundation.  Additional servers can be procured on behalf of the project pending request and budget approval.
+
+## DNS
+The OpenJS Foundation will register and manage each project’s primary domain, and the .org, .dev, .net, and .com variants (as available).  The project’s primary domain must be transferred to the Foundation if the project desires the foundation to take financial and management responsibility for the domain.  Management of additional domain names may be requested, subject to budget approval.
+
+The OpenJS Foundation can either manage your DNS for you, or delegate to one of your own nameservers.  This is often required if you use a CDN (see below).
+
+## SSL Certificates
+The OpenJS Foundation will purchase 2-year wildcard SSL certs for each projects’ managed domains as needed.
+
+## Source Control
+By default all OpenJS Foundation projects have open source repositories in their own GitHub Organizations.  The OpenJS Foundation admin account must be added as administrator for each repository.  Two-factor authentication must be required for everyone in the organization.
```

Comment:
> The OpenJS Foundation admin account must be added as administrator for each repository.

This bit also belong in the onboarding checklist, no?

Reply:
indeed, good catch

Reply:
I captured this in #404; resolving here